tests/e2e/util.go

```diff
@@ -6138,3 +6155,29 @@ func readDataFromRawBlockVolume(ns string, podName string, devicePath string, te
 	framework.ExpectNoError(err, fmt.Sprintf("failed to read device: %q inside the pod: %q", devicePath, podName))
 	gomega.Expect(strings.Contains(output, testData)).NotTo(gomega.BeFalse())
 }
+
+// verifyClusterIdConfigMapGeneration verifies if cluster id configmap gets generated by
+// csi driver in csi namespace
+func verifyClusterIdConfigMapGeneration(client clientset.Interface, ctx context.Context,
+	csiNamespace string, configMapName string, cmToExist bool) {
+	_, err := client.CoreV1().ConfigMaps(csiNamespace).Get(ctx, configMapName,
+		metav1.GetOptions{})
+	if cmToExist && apierrors.IsNotFound(err) {
+		framework.Logf("Configmap: %s not found in namespace: %s", configMapName, csiNamespace)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	} else if !cmToExist && !apierrors.IsNotFound(err) {
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+}
+
+// fetchClusterIdFromConfigmap fetches cluster id value from
+// auto generated cluster id configmap by csi driver
+func fetchClusterIdFromConfigmap(client clientset.Interface, ctx context.Context,
+	csiNamespace string, configMapName string) string {
+	clusterIdCm, err := client.CoreV1().ConfigMaps(csiNamespace).Get(ctx, configMapName,
```

Comment:
why are we getting`configMapName`, it should be a constant AFAIK

Reply:
Fixed it